rustls/src/error.rs

```diff
@@ -262,6 +262,19 @@ pub enum CertificateError {
     Other(Arc<dyn StdError + Send + Sync>),
 }
 
+impl From<CertificateError> for AlertDescription {
```

Comment:
Would be good to add the OpenSSL and BoringSSL links from the commit message here as comments, probably!